content/working_with/console/widgets/sitesMap.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bt_wiki
+title: Sites Map
+category: Cloudify Console
+draft: false
+---
+Displays the world map with defined sites marked. Only sites with defined location are displayed on the map.
+
+![sitesMap]( /images/ui/widgets/sitesMap.png )
+
+Color of the site marker on the map indicates site health. It is derived from status of deployments attached to the site.
+ 
+After clicking on site marker on the map, user can see popup with site details.
+
+![sitesMap]( /images/ui/widgets/sitesMap_siteDetails.png )
+
+Site details contains information about deployments' statuses, indicated as follows:
```

Comment:
Since 'site details' are plural it should be 'Site details contain'

Reply:
👍